### ISSUE TITLE
Convergence

### ISSUE THREAD

**Issue by MathiasKraus** [johnb30/py_crepe#6]
Hi,
thank you for this awesome keras implementation! I just tried to run the code example and the model doesn't seem to converge. After 7 epochs, the loss is:
Loss: 1.38647970024. Accuracy: 0.250000002667
I obtained the AG-news dataset from https://github.com/mhjabreel/CharCNN. The xt, yt matrices seem to be ok. Can someone recheck, if the model is converging? I tried both backends, tensorflow and theano with the same result.
Thanks in advance!

**Comment by liufuyang** [comment]
Have you solved your problem? I encountered the same issue.
